otherlibs/stdune/src/copyfile_stubs.c

```diff
@@ -73,7 +73,7 @@ CAMLprim value stdune_copyfile(value v_from, value v_to) {
   caml_failwith("copyfile: only on macos");
 }
 
-static int dune_sendfile(int in, int out, int length) {
+static int dune_sendfile(int in, int out, size_t length) {
   int ret;
```

Comment:
```suggestion
  ssize_t ret;
```

(since that's the return type of `sendfile`)